extensions/amp-script/amp-script.md

```diff
@@ -142,9 +214,13 @@ AMP.getState(expr) {}
 </script>
 ```
 
-### Exporting a function for use in `<amp-list>`
+[tip type="default"]
+To use `AMP.setState()`, you must include the [`amp-bind`](https://amp.dev/documentation/components/amp-bind) extension script in the document head.
+[/tip]
+
+### Retrieving data for `<amp-list>`
 
-You may export a function to act as the data source for an `<amp-list>`.
+You may export a function to act as the data source for an [`<amp-list>`](/documentation/components/amp-list/).
```

Comment:
this link too @matthiasrohmer

Reply:
works for me in my dev environment, but ymmv

Reply:
Made absolute as per our conversation.